packages/react-inbox/src/components/Message/index.tsx

```diff
@@ -71,22 +72,18 @@ const Message: React.FunctionComponent<MessageProps> = ({
     () =>
       [
         showMarkAsRead && {
-          label: "Mark as Read",
+          label: MESSAGE_LABELS.MARK_AS_READ,
```

Comment:
if these labels are only used in one place, does making constants file do anything for us or is it just something "to do"?

Reply:
my personal persepctive is that if something is used >1 or maybe event >2 times then abstract it out... but if its only a one time constant or a a one time peice of logic than writing a separate file/function just adds unnecesary complexity

Reply:
my opinion though, so feel free to disregard

Reply:
cool cool, just sharing my thoughts.  we should add this to the list of things we want to do as a part of our contributing guide we will someday make.  me you and @suhasdeshpande  should probably have a meeting to come up with some best practices